tests/all_mode/conf/dm-task.yaml

```diff
@@ -13,6 +13,11 @@ target-database:
   port: 4000
   user: "root"
   password: ""
+  session:
+    sql_mode: "ONLY_FULL_GROUP_BY,STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION"
+    tidb_skip_utf8_check: 1
+    tidb_disable_txn_auto_retry: off
+    tidb_retry_limit: "10"
```

Comment:
Is there any method to verify any or all of them have taken effect?

Reply:
Unit test need launch tidb, so I add test in integration test in 9ec51cb.